cookbook/doctrine/registration_form.rst

```diff
@@ -135,7 +132,7 @@ example, see the :ref:`Entity Provider <security-crete-user-entity>` article.
 
 .. sidebar:: Why the 4096 Password Limit?
 
-    Notice that the ``plainPassword`` field has a max length of 4096 characters.
+    Notice that the ``password`` field has a max length of 4096 characters.
```

Comment:
This whole paragraph must be reworded to reflect the fact that we only store the hashed (encrypted) password.